rskj-core/src/main/java/co/rsk/crypto/Keccak256.java

```diff
@@ -55,6 +55,11 @@ public String toJsonString() {
         return HexUtils.toUnformattedJsonHex(this.bytes);
     }
 
+    public String toHexString(boolean addPrefix) {
```

Comment:
If only it is going to be used on test, not sure if we should have this method here, maybe can be used wrongly. 
Also there is an method in HexUtils to add the prefix if needed `co.rsk.util.HexUtils.ToJsonHex(String param)`